frontend/javascripts/dashboard/advanced_dataset/dataset_table.tsx

```diff
@@ -278,24 +278,45 @@ class DatasetRenderer {
     return <FileOutlined style={{ fontSize: "18px" }} />;
   }
   renderNameColumn() {
+    let selectedLayerName = null;
+    if (this.data.colorLayerNames.length > 0) {
+      selectedLayerName = this.data.colorLayerNames[0];
+    } else if (this.data.segmentationLayerNames.length > 0) {
+      selectedLayerName = this.data.segmentationLayerNames[0];
+    }
+    const imgSrc = selectedLayerName
+      ? `/api/datasets/${this.data.owningOrganization}/${this.data.name}/layers/${selectedLayerName}/thumbnail?w=200&h=200`
```

Comment:
is it intentional that the thumbnails are requested in 200x200, but there are only rendered in 80x80? is this good for high dpi screens?

Reply:
Almost intentional 😅 I had it 100×100 in CSS first, with 200×200 graphics, allowing for screens with px-to-device-pixel factor of 2, which is the most common currently, other than 1.
As I changed the css size to 80×80, I’ll adapt this to use 160×160 then :)

Reply:
How about defining a module constant which is then used in the url as well as in an inline `style`? :) something like this:

```
const THUMBNAIL_EXTENT = 100;

`...thumbnail?w=${THUMBNAIL_EXTENT}&h=${THUMBNAIL_EXTENT}`
...<img style={{width: 2 * THUMBNAIL_EXTENT}}...
```

Reply:
in the top level scope below the imports is fine :)